src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -137,12 +137,12 @@ ClientBackend::Create(
         url, protocol, ssl_options, BackendToGrpcType(compression_algorithm),
         http_headers, verbose, &local_backend));
   } else if (kind == TENSORFLOW_SERVING) {
-    RETURN_IF_CB_ERROR(tfserving::TFServeClientBackend::Create(
-        url, protocol, BackendToGrpcType(compression_algorithm), http_headers,
-        verbose, &local_backend));
+    // RETURN_IF_CB_ERROR(tfserving::TFServeClientBackend::Create(
```

Comment:
Uncomment these lines here and below for TFS and torch serve.

Reply:
Uncommented 👍